src/devices/Ssd1351/ssd1351Imaging.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Runtime.InteropServices;
+
+using Iot.Device.Ssd1351.Command;
+
+namespace Iot.Device.Ssd1351
+{
+    /// <summary>
+    /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+    /// light emitting diode dot-matrix graphic display system. 
+    /// </summary>
+    public partial class Ssd1351 : IDisposable
+    {
+
+        /// <summary>
+        /// Send a bitmap to the ssd1351 display specifying the starting position and destination clipping rectangle.
+        /// </summary>
+        /// <param name="bm">The bitmap to be sent to the display controller note that only Pixel Format Format32bppArgb is supported.</param>
+        public void SendBitmap(Bitmap bm)
+        {
+            SendBitmap(bm, new Point(0, 0), new Rectangle(0, 0, SCREEN_WIDTH_PX, SCREEN_WIDTH_PX));
+        }
+
+        /// <summary>
+        /// Send a bitmap to the ssd1351 display specifying the starting position and destination clipping rectangle.
+        /// </summary>
+        /// <param name="bm">The bitmap to be sent to the display controller note that only Pixel Format Format32bppArgb is supported.</param>
+        /// <param name="destinationRect">A rectangle that defines where in the display the bitmap is written. Note that no scaling is done.</param>
+        public void SendBitmap(Bitmap bm, Rectangle updateRect)
+        {
+            SendBitmap(bm, new Point(updateRect.X, updateRect.Y), updateRect);
+        }
+
+        /// <summary>
+        /// Send a bitmap to the ssd1351 display specifying the starting position and destination clipping rectangle.
+        /// </summary>
+        /// <param name="bm">The bitmap to be sent to the display controller note that only Pixel Format Format32bppArgb is supported.</param>
+        /// <param name="sourcePoint">A coordinate point in the source bitmap where copying starts from.</param>
+        /// <param name="destinationRect">A rectangle that defines where in the display the bitmap is written. Note that no scaling is done.</param>
+        public void SendBitmap(Bitmap bm, Point sourcePoint, Rectangle destinationRect)
+        {
+
```

Comment:
nit: extra enter (not commenting anymore)

Reply:
Not sure what rule is being broken here but I have removed the line after the brace.